js/5etools-monsters.js

```diff
@@ -1120,6 +1120,27 @@ function d20plusMonsters () {
 									character.attribs.create({name: `repeating_npctrait_${newRowId}_desc`, current: text});
 								});
 							}
+							if (data.variant && d20plus.cfg.getOrDefault("import", "importVariants")) {
+								$.each(data.variant, function (i, v) {
+									let newRowId = d20plus.ut.generateRowId();
+									character.attribs.create({
+										name: `repeating_npctrait_${newRowId}_name`,
+										current: d20plus.importer.getCleanText(renderer.render('Variant : '+v.name)),
+									});
+
+									if (d20plus.cfg.getOrDefault("import", "importVariants")) {
+										const offsetIndex = data.spellcasting ? 1 + i : i;
```

Comment:
`data.spellcasting` is an array which can be >1 item long; why not `const offsetIndex = (data.spellcasting?.length || 0)+ i;`?

Reply:
Fixed, most code was copy pasted from 1102-1122 , import traits code. should i change those too?